Docker/run-with-events.sh

```diff
@@ -1,16 +1,32 @@
 #!/bin/bash
 
+script_path=$(dirname $(readlink -f $0))
+
+if [[ "$1" == "cleanup" ]]; then
+    docker-compose -p cyberway -f $script_path/docker-compose-events.yml down || exit 1
+    docker volume rm cyberway-mongodb-data
+    docker volume rm cyberway-nodeos-data
+    docker volume rm cyberway-queue
+    exit 0
+elif [[ "$1" == "up" ]]; then
+    docker-compose -p cyberway -f $script_path/docker-compose-events.yml up -d || exit 1
+    exit 0
+elif [[ "$1" == "down" ]]; then
+    docker-compose -p cyberway -f $script_path/docker-compose-events.yml down || exit 1
+    exit 0
+fi
+
 docker volume create cyberway-mongodb-data || true
 docker volume create cyberway-nodeos-data || true
-docker volume create cyberway-keosd-data || true
 docker volume create cyberway-queue || true
 
-EXPORTER_PASS=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
-EXPORTER_USER=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 8 | head -n 1)
-
-if [[ ! -e .env ]]; then
-  echo "EXPORTER_USER=$EXPORTER_USER" >> .env
-  echo "EXPORTER_PASS=$EXPORTER_PASS" >> .env
-fi 
+if [[ ( -z "$NATS_USER" ) || ( -z "$NATS_PASS" ) ]]; then
+    if [[ ! -e .env ]]; then
```

Comment:
Can't get the logic of this check. If we have no NATS_USER or NATS_PASS AND .env does not exist then... create .env with random entries. But NATS_USER and NATS_PASS still remains empty after scripts exit, so why do we check them here?

Reply:
The user can define values for these variables in environment variables. If the user did not do this, then we prepare the .env file that the docker-compose will use.
You added a check for the presence of these variables in docker-compose script, so we can remove this logic and leave the user the ability to initialize the .env file himself. I'll do it in the next task.